### PR TITLE
Clean up fb

### PR DESCRIPTION
**Pull request by Ian-Stewart-Binks**
This pull request cleans up the Facebook code that was recently pulled into master in #489.

Note that I have also made a change in `graph/node.js`. Before, the code was checking the logical type against both `or` and `OR` (and `and` and `AND`). The code now converts the logical type to uppercase and compares it against `AND`.

### CODE REVIEW

File: public/js/grid/timetable_util.js [hunk 1]
```diff
@@ -175,13 +174,13 @@ function saveCookies(courses, sections) {
     'use strict';
 
     if (courses !== undefined) {
-        setCookie("selected-courses", JSON.stringify(courses).replace(/,/g, '_').replace(/[\"\[\]]/g, ''));
+        setCookie("selected-courses", courses.join('_').replace(/[\"\[\]]/g, ''));
```
Comment (david-yz-liu) [inline public/js/grid/timetable_util.js]: I don't know that the `replace` is necessary any more - did you check?

Comment (Ian-Stewart-Binks) [inline public/js/grid/timetable_util.js]: It doesn't look like it, but when trying it out I've come across what appears to be an unrelated bug...

**Comment by david-yz-liu** [comment]
Thanks, @Ian-Stewart-Binks!
